sagemaker-remote-function/quick_start/quick_start.ipynb

```diff
@@ -0,0 +1,281 @@
+{
```

Comment:
<p>Maybe a more specific title than "Indroduction" (i.e. introduction to what?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>done</p>